src/components/AddCourseDropdown.jsx

```diff
@@ -2,24 +2,26 @@ import React, { PropTypes } from 'react';
 import { FormGroup, ControlLabel, FormControl } from 'react-bootstrap';
 
 const AddCourseDropdown = ({ onSelect, catId, category, courses, selected }) => (
-  <FormGroup controlId={catId}>
-    <ControlLabel>{category}</ControlLabel>
-    <FormControl componentClass="select" value={selected} onChange={onSelect}>
-      <option key={' '} value={' '} />
-      {courses.map(course =>
-        <option key={course.courseId} value={course.courseId}>{course.name}</option>,
-      )}
-    </FormControl>
-  </FormGroup>
+  <li>
+    <FormGroup controlId={catId}>
```

Comment:
What is "catId"? Full words in the variable names for clarity, please. Here and elsewhere.